hw/ip/spi_host/dv/spi_host_sim_cfg.hjson

```diff
@@ -30,6 +30,8 @@
                 // Common CIP test lists
                 "{proj_root}/hw/dv/tools/dvsim/tests/csr_tests.hjson",
                 "{proj_root}/hw/dv/tools/dvsim/tests/mem_tests.hjson",
+                "{proj_root}/hw/dv/tools/dvsim/tests/alert_test.hjson",
+                "{proj_root}/hw/dv/tools/dvsim/tests/intr_test.hjson",
```

Comment:
@tunghoang290780 looks like the interrupt auto test is missing here. Please let me know if you have any concerns.